octobot_commons/configuration/configuration.py

```diff
@@ -33,21 +33,21 @@ class Configuration:
     """
 
     def __init__(
-        self,
-        config_path: str,
-        profiles_path: str,
-        schema_path: str = None,
-        profile_schema_path: str = None,
+            self,
```

Comment:
black wont like it

Reply:
sorry, pycharm do it automatically. it's fixed now